src/schema/v2/Alerts/index.ts

```diff
@@ -177,157 +177,190 @@ export const AlertType = new GraphQLObjectType<
   ResolverContext
 >({
   name: "Alert",
-  fields: () => ({
-    ...IDFields,
-    acquireable: {
-      type: GraphQLBoolean,
-    },
-    additionalGeneIDs: {
-      type: new GraphQLList(GraphQLString),
-      resolve: ({ additional_gene_ids }) => additional_gene_ids,
-    },
-    additionalGeneNames: {
-      type: new GraphQLList(GraphQLString),
-      resolve: ({ additional_gene_names }) => additional_gene_names,
-    },
-    artistIDs: {
-      type: new GraphQLList(GraphQLString),
-      resolve: ({ artist_ids }) => artist_ids,
-    },
-    artists: {
-      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(ArtistType))),
-      resolve: async ({ artist_ids }, _args, { artistsLoader }) => {
-        if (!artist_ids) return []
+  fields: () => {
```

Comment:
most of the changes in this file are caused by the shift of everything inside `fields` function by 1 tab. The only real change is new `artworksConnection`